subprojects/scala/src/integTest/groovy/org/gradle/scala/scaladoc/ScalaDocIntegrationTest.groovy

```diff
@@ -83,4 +83,28 @@ class ScalaDocIntegrationTest extends AbstractIntegrationSpec implements Directo
         // Started Gradle worker daemon (0.399 secs) with fork options DaemonForkOptions{executable=/Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home/bin/java, minHeapSize=null, maxHeapSize=234M, jvmArgs=[], keepAliveMode=DAEMON}.
         outputContains("maxHeapSize=234M")
     }
+
+    def "scaladoc uses scala3"() {
```

Comment:
Is there anything specific to Scala 3 scaladoc that we could test for here?

Reply:
I'll think about that, but I think the output is very alike to other doc tools. We can assert the existence of certain files I think.